sopt/src/main/scala/dagr/sopt/cmdline/SpecialArgumentsCollection.scala

```diff
@@ -21,33 +21,22 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.cmdline.parsing
+package dagr.sopt.cmdline
 
-import dagr.core.cmdline.Arg
+import dagr.sopt._
 
-private[parsing] object SpecialArgumentsCollection {
+private[cmdline] object SpecialArgumentsCollection {
   val HELP_FULLNAME: String = "help"
   val VERSION_FULLNAME: String = "version"
-  //val ARGUMENTS_FILE_FULLNAME: String = "arguments-file"
 }
 
 /**
   * This collection is for arguments that require special treatment by the arguments parser itself.
   * It should not grow beyond a very short list.
-  *
-  * An arguments file will split apart by spaces, but all neighbouring strings without a leading
-  * dash ('-') will be grouped.  Use multiple lines to ungroup the arguments
-  *   Example 1:
-  *   File: "--aab s --b -c -d wer we -c"
-  *   Args: "--aab" "s" "--b" "-c" "-d" "wer we" "-c"
-  *
-  *   Example 2:
-  *   File: "--aab s --b -c -d wer\nwe -c"
-  *   Args: "--aab" "s" "--b" "-c" "-d" "wer", "we" "-c"
   */
-private[parsing] final case class SpecialArgumentsCollection(
-  @Arg(flag = "h", name = "help", doc = "Display the help message.", special = true)
+private[cmdline] final case class SpecialArgumentsCollection(
+  @arg(flag = "h", name = "help", doc = "Display the help message.", special = true)
```

Comment:
Not new to this PR, but I still don't love the way the special arguments work.  Would be nice to find a tidier way to do this.